docs/SECURITY.md

```diff
@@ -0,0 +1,25 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+To report a vulnerability, please contact the Security Team: `cncf-kubeedge-security@lists.cncf.io`. Security is of the highest importance and we sincerely request you to keep the vulnerability information confidential.
```

Comment:
We can add some template for the email content here. And remindering the user don't report vulnerability to public like issue or slack msg.

Reply:
using bug report template, and report to cncf-kubeedge-security@lists.cncf.io